docs/content/en/docs/getting_started.md

```diff
@@ -432,3 +432,22 @@ bazel run --noallow_analysis_cache_discard //tools:heir-opt -- \
 --insert-rotate --cse --canonicalize --collapse-insertion-chains \
 --canonicalize --cse /path/to/heir/tests/simd/box_blur_64x64.mlir
 ```
+
+### Optional: Graphviz visualization of the IR
+
+Getting a visualization of the IR during optimization/transformation might help
+you understand what is going on more easily.
+
+Still taking the `dot_product_8.mlir` as an example:
+
+```bash
+bazel run --ui_event_filters=-info,-debug,-warning,-stderr,-stdout --noshow_progress --logging=0 //tools:heir-opt -- --wrap-generic --heir-simd-vectorizer $PWD/tests/Examples/openfhe/dot_product_8.mlir --view-op-graph 2> dot_product_8.dot
```

Comment:
(review from my phone since I'm not working 😁) To make this simpler, I'm guessing you can use `-o` to write the output to a file, which would remove the need for all the bazel quieting commands.

Reply:
There was no `-o`-like option for `bazel run`. There was a `--output-file` for `bazel query` but that only redirects the stdout instead of stderr.

Links
 * https://bazel.build/reference/command-line-reference
 * https://github.com/bazelbuild/bazel/issues/5511